src/Microsoft.ML.TimeSeries/SrCnnDataType.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Transforms.TimeSeries
+{
+    /// <summary>
+    /// The detect modes of SrCnn models.
+    /// </summary>
+    public enum SrCnnDetectMode
+    {
+        /// <summary>
+        /// In this mode, output (IsAnomaly, RawScore, Mag).
+        /// </summary>
+        AnomalyOnly = 0,
+
+        /// <summary>
+        /// In this mode, output (IsAnomaly, AnomalyScore, Mag, ExpectedValue, BoundaryUnit, UpperBoundary, LowerBoundary).
+        /// </summary>
+        AnomalyAndMargin = 1,
+
+        /// <summary>
+        /// In this mode, output (IsAnomaly, RawScore, Mag, ExpectedValue).
+        /// </summary>
+        AnomalyAndExpectedValue = 2
+    }
+
+    /// <summary>
+    /// Timeseries point definition for SrCnn models.
+    /// </summary>
+    public sealed class SrCnnTsPoint
+    {
+        public DateTime Timestamp { get; set; }
+        public Double Value { get; set; }
+
+        public SrCnnTsPoint(DateTime timestamp, Double value)
+        {
+            Timestamp = timestamp;
+            Value = value;
+        }
+    }
+
+    /// <summary>
+    /// Dataview type of <see cref="SrCnnTsPoint"/>.
+    /// </summary>
+    public sealed class SrCnnTsPointDataViewType : StructuredDataViewType
+    {
```

Comment:
Why do we need this? And what is it being used for?

Reply:
> Why do we need this? And what is it being used for?

Removed since no need for SrCnnTsPoint now.